commands/command_event.h

```diff
@@ -57,6 +59,10 @@ class AddEventDivemodeSwitch : public AddEventBase {
 class AddEventSetpointChange : public AddEventBase {
 public:
 	AddEventSetpointChange(struct dive *d, int dcNr, int seconds, pressure_t pO2);
+private:
+	divemode_t divemode;	// Wonderful: this may change the divemode of the dive to CCR
```

Comment:
It does because historically, there was no divemode, a dive was a CCR dive by definition if it had an event setting a non-zero set-point anywhere.